quinn-h3/src/server.rs

```diff
@@ -83,11 +203,25 @@ impl Builder {
         Ok(self)
     }
 
+    /// Set H3 settings.
     pub fn settings(&mut self, settings: Settings) -> &mut Self {
         self.settings = settings;
         self
     }
 
+    /// Specify a base Quic configuration.
```

Comment:
```suggestion
    /// Specify a base QUIC configuration.
```

(Here and elsewhere.)